08-projects.Rmd

```diff
@@ -6453,24 +6453,14 @@ Make sure to replace "\<title id here\>" with the `title_id` of your favorite sh
 
 #### Dataset 
 
-The following questions will use the dataset found in Scholar:
+The following questions will use PLOTSNAP.csv from the data folder found in Scholar:
```

Comment:
It seems that TREE.csv is not used in the 290 Project 14. So I removed the relevant links.

Reply:
I agree thank you!